src/resources/questions-animal-and-pets/question-4/question.html

```diff
@@ -1,3 +1,10 @@
-<p>
-<b>  <span class="special">*</span> Do you allow animals in outdoor dining areas?</b>
+<p class="question-margin">
```

Comment:
how do all of these small files fit together?

Reply:
Our build process.